dev-environment.yml

```diff
@@ -30,3 +30,6 @@ dependencies:
   - cython
   - panel
   - bokeh<2.3  # temp restriction until panel/bokeh bugs are worked out
+  - cmake >= 3.13.4
+  - conda-forge::ninja
```

Comment:
Is there a reason we need ninja from conda-forge?  It looks like the same versions are available in defaults as well.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/38/commits/053d56814992458df38906aa914637ea579ec2f6 removes the use of `conda-forge` for `ninja`.